README.MD

```diff
@@ -36,6 +32,7 @@ Contributions, issues, and feature requests are welcome!
 Give a ⭐️ if you like this project!
 
 ## Acknowledgments
+> Image Credits to. (istockphoto.com) 
 
 - Hat tip to anyone who's code/resource was used
 - Inspiration
```

Comment:
- [ ] In your README file kindly provide the `Acknowledgement` that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).